src/main/user-impl/java/com/mysql/cj/jdbc/NonRegisteringDriver.java

```diff
@@ -196,7 +204,8 @@ public boolean acceptsURL(String url) throws SQLException {
 
             ConnectionUrl conStr = ConnectionUrl.getConnectionUrlInstance(url, info);
             Map<String, String> connProps = conStr.getOriginalProperties();
-            boolean acceptAwsProtocolOnly = Boolean.parseBoolean(connProps.getOrDefault(PropertyKey.acceptAwsProtocolOnly.getKeyName(), "false"));
+            acceptAwsProtocolOnly = Boolean.parseBoolean(connProps.getOrDefault(PropertyKey.acceptAwsProtocolOnly.getKeyName(), "false"));
```

Comment:
I think there are a couple potential problems here. Seems like there's two levels at which we can set this attribute: the connection level for just this specific connection or the driver level for every single connection. I think if we set it as a connection parameter it shouldn't modify the driver-level attribute. Also, instead of defaulting to "false", shouldn't we default to the driver-level value? We want to use the connection parameter when present, but if it isn't, then use the driver-level value

Reply:
We might have a different understanding of the use case of these switches, following up offline.

Reply:
I think our current understanding is that we are providing two different ways to set the same Driver level variable. If this changes that part should be modified as discussed.

Reply:
It changed and that part is adjusted :)